src/Ilios/AuthenticationBundle/RelationshipVoter/CurriculumInventoryAcademicLevel.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Ilios\AuthenticationBundle\RelationshipVoter;
+
+use Ilios\AuthenticationBundle\Classes\SessionUserInterface;
+use Ilios\CoreBundle\Entity\CurriculumInventoryAcademicLevelInterface;
+use Ilios\CoreBundle\Entity\DTO\CurriculumInventoryAcademicLevelDTO;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+
+class CurriculumInventoryAcademicLevel extends AbstractVoter
+{
+    protected function supports($attribute, $subject)
+    {
+        return (
+            ($subject instanceof CurriculumInventoryAcademicLevelDTO && in_array($attribute, [self::VIEW])) or
+            ($subject instanceof CurriculumInventoryAcademicLevelInterface && in_array($attribute, [self::VIEW]))
```

Comment:
Why is this read only? Can't it be updated by anyone who can update a report?

Reply:
read-only is correct in this case. these are generated when the report itself is created, and are fixed to ten levels each per report.